bin/oref0-log-shortcuts.sh

```diff
@@ -9,18 +11,18 @@ grep -q autotunelog ~/.bash_profile || echo "alias autotunelog="'"tail -n 100 -F
 grep -q pump-looplog ~/.bash_profile || echo "alias pump-looplog="'"tail -n 100 -F /var/log/openaps/pump-loop.log"' >> ~/.bash_profile
 grep -q urchin-looplog ~/.bash_profile || echo "alias urchin-looplog="'"tail -n 100 -F /var/log/openaps/urchin-loop.log"' >> ~/.bash_profile
 grep -q ns-looplog ~/.bash_profile || echo "alias ns-looplog="'"tail -n 100 -F /var/log/openaps/ns-loop.log"' >> ~/.bash_profile
-grep -q cat-pref ~/.bash_profile || echo "alias cat-pref="'"cd ~/myopenaps && cat preferences.json"' >> ~/.bash_profile
-grep -q edit-pref ~/.bash_profile || echo "alias edit-pref="'"cd ~/myopenaps && nano preferences.json"' >> ~/.bash_profile
+grep -q cat-pref ~/.bash_profile || echo "alias cat-pref=\"cd $myopenaps && cat preferences.json\"" >> ~/.bash_profile
+grep -q edit-pref ~/.bash_profile || echo "alias edit-pref=\"cd $myopenaps && nano preferences.json\"" >> ~/.bash_profile
 grep -q cat-wifi ~/.bash_profile || echo "alias cat-wifi="'"cat /etc/wpa_supplicant/wpa_supplicant.conf"' >> ~/.bash_profile
 grep -q edit-wifi ~/.bash_profile || echo "alias edit-wifi="'"nano /etc/wpa_supplicant/wpa_supplicant.conf"' >> ~/.bash_profile
-grep -q cat-runagain ~/.bash_profile || echo "alias cat-runagain="'"cd ~/myopenaps && cat oref0-runagain.sh"' >> ~/.bash_profile
-grep -q edit-runagain ~/.bash_profile || echo "alias edit-runagain="'"cd ~/myopenaps && nano oref0-runagain.sh"' >> ~/.bash_profile
-grep -q cat-autotune ~/.bash_profile || echo "alias cat-autotune="'"cd ~/myopenaps/autotune && cat autotune_recommendations.log"' >> ~/.bash_profile
+grep -q cat-runagain ~/.bash_profile || echo "alias cat-runagain=\"cd $myopenaps && cat oref0-runagain.sh\"" >> ~/.bash_profile
+grep -q edit-runagain ~/.bash_profile || echo "alias edit-runagain=\"cd $myopenaps && nano oref0-runagain.sh\"" >> ~/.bash_profile
+grep -q cat-autotune ~/.bash_profile || echo "alias cat-autotune=\"cd ${myopenaps}/autotune && cat autotune_recommendations.log\"" >> ~/.bash_profile
```

Comment:
Why curly braces here but not the prior ones?  We should probably make them all consistent.